Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 
 [dependencies]
 anyhow = "1"
-async-graphql = "7"
+async-graphql = { version = "7", features = ["chrono"] }
```

Comment:
The `chrono` feature is currently omitted from the `main` branch (due to usage of `Jiff` crate). Please update the `Cargo.toml` and `Cargo.lock` files accordingly so that your branch can be rebased onto `main`.